Add new docs: abbreviations and tone of voice (#7)
vtexdocs/language-hub-content: docs/en/grammar/abbreviations.md

---
title: 'Abbreviations'
id: grammar0006
status: PUBLISHED
createdAt: 2025-02-10T19:24:22.388Z
updatedAt: 2025-02-10T19:24:22.388Z
publishedAt: 2025-02-10T19:24:22.388Z
firstPublishedAt: 2025-02-10T19:24:22.388Z
contentType: trackArticle
productTeam: Localization
slugEN: abbreviations
locale: en
trackId: 5PxyAgZrtiYlaYZBTlhJ2A
trackSlugEN: abbreviations
---

# Abbreviations include acronyms, shortened words, and contractions.

In documentation, spell out abbreviations on the first reference if they’re not common and include the abbreviation in parentheses immediately after.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • The `Filter` component is used to filter products inside the *Product Listing Page (PLP)* and Search Page. | • The `Filter` component is used to filter products inside the *PLP* and Search Page. |

Reviewer: ```suggestion
| The `Filter` component is used to filter products inside the *Product Listing Page (PLP)* and Search Page. | The `Filter` component is used to filter products inside the *PLP* and Search Page. |
```

Don’t use *i.e.* or *e.g.* They are not localization friendly.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • It is frequently necessary to send the item price to the gateway or to your ERP with the tax shown separately from the price. <br> • In this function, create an event object specifying the desired event type (example: ‘purchase’) and then call ‘sendAnalyticsEvent’. | • It’s frequently necessary to send the item price to the Gateway or to your ERP with the tax shown, i.e. separately from the price. <br> • In this function, create an event object specifying the desired event type (e.g., ‘purchase’) and then call ‘sendAnalyticsEvent’. |

Reviewer: ```suggestion
| <ol><li> It is frequently necessary to send the item price to the gateway or to your ERP with the tax shown separately from the price. </li><li> In this function, create an event object specifying the desired event type (example: ‘purchase’) and then call ‘sendAnalyticsEvent’. </li></ol>| <ol><li> It’s frequently necessary to send the item price to the Gateway or to your ERP with the tax shown, i.e. separately from the price.</li><li>In this function, create an event object specifying the desired event type (e.g., ‘purchase’) and then call ‘sendAnalyticsEvent’.</li></ol> |
```

## Periods with abbreviations

Don't use periods with acronyms or the abbreviation for the name of a country.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • You need to send the item price to your ERP with the tax shown separately from the price. <br> • This amount is shown in US dollars. | • You need to send the item price to your E.R.P. with the tax shown separately from the price. <br> • This amount is shown in U.S. dollars. |

Reviewer: ```suggestion
| <ol><li>You need to send the item price to your ERP with the tax shown separately from the price.</li><li> This amount is shown in US dollars.</li></ol> | <ol><li>You need to send the item price to your E.R.P. with the tax shown separately from the price.</li><li>This amount is shown in U.S. dollars.</li></ol> |
```

If you write or say an abbreviation as a word (for example, app or sync), don't put a period after it.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • The app is available in the Extensions Hub. <br> • You can sync your catalog with Amazon. | • The app. is available in the Extensions Hub. <br> • You can sync. your catalog with Amazon. |

Reviewer: ```suggestion
| <ol><li>The app is available in the Extensions Hub.</li><li>You can sync your catalog with Amazon.</li></ol> | <ol><li>The app. is available in the Extensions Hub.</li><li> You can sync. your catalog with Amazon.</li></ol> |
```

Put a period at the end of a shortened word in text, including strings that are full phrases. Omit for date and time, forms, and column headers in the UI.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • Check the item qty. in the cart | • Check the item qty in the cart <br> • No. |

Reviewer: ```suggestion
| Check the item qty. in the cart | <ol><li>Check the item qty in the cart</li><li>No.</li></ol> |
```

### Contractions

Always use common contractions, such as *it’s*, *you’re*, *that's, don’t, can’t, you’ll.* Contractions help avoid misreading, as it's easier for readers to overlook the word *not* in a sentence than to misinterpret a contraction like *don’t* as *do*. 

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • You don't have any manually added products. | • You do not have any manually added products. |

Reviewer: ```suggestion
| You don't have any manually added products. | You do not have any manually added products. |
```

Don’t mix contractions and spelled-out equivalents within the same sentence.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • **It’s** key for data encoding to be part of this routine, but **it's** important noting that creating cookies can also interfere with the store in other ways. | • **It is** key for data encoding to be part of this routine, but **it’s** important noting that creating cookies can also interfere with the store in other ways. |

Reviewer: ```suggestion
| **It’s** key for data encoding to be part of this routine, but **it's** important noting that creating cookies can also interfere with the store in other ways. | **It is** key for data encoding to be part of this routine, but **it’s** important noting that creating cookies can also interfere with the store in other ways. |
```

Avoid contractions formed from nouns and verbs.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • VTEX is continuously evolving our products. | • VTEX’s continuously evolving our products. |

Reviewer: ```suggestion
| VTEX is continuously evolving our products. | VTEX’s continuously evolving our products. |
```

Don’t use double contractions like *mustn't've* or *shouldn't've*, and avoid ambiguous or awkward contractions, such as *there’d*, *it’ll*, and *they’d.*

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • It will disappear from the list on the SKU Bindings page. | • It'll disappear from the list on the SKU Bindings page. |

Reviewer: ```suggestion
| It will disappear from the list on the SKU Bindings page. | It'll disappear from the list on the SKU Bindings page. |
```

###  Acronyms 

Most acronyms are fully capitalized, but not all. Follow the standard form used in the industry. Capitalize all words in the spelled-out form of an acronym. The names of many protocols and specifications are considered proper nouns and are capitalized when spelled out.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • Renew the API token before the duration ends. <br> • This option involves using a third-party platform, known as Platform as a Service (PaaS). <br> • How does VTEX ensure compliance with the General Data Protection Regulation (GDPR)? <br> • Multiple redirects are possible in a single URL. <br> • A user ID screen is displayed so that the customer can continue trying to purchase.  | • Renew the api token before the duration ends. <br> • This option involves using a third-party platform, known as platform as a service (PAAS). <br> • How does VTEX ensure compliance with the general data protection regulation (GDPR)? <br> • Multiple redirects are possible in a single url. <br> • A user id screen is displayed so that the customer can continue trying to purchase. |

Reviewer: ```suggestion
| <ol><li>Renew the API token before the duration ends.</li><li>This option involves using a third-party platform, known as Platform as a Service (PaaS).</li><li> How does VTEX ensure compliance with the General Data Protection Regulation (GDPR)?</li><li> Multiple redirects are possible in a single URL.</li><li> A user ID screen is displayed so that the customer can continue trying to purchase.</li></ol>  | <ol><li>Renew the api token before the duration ends.</li><li> This option involves using a third-party platform, known as platform as a service (PAAS).</li><li> How does VTEX ensure compliance with the general data protection regulation (GDPR)?</li><li> Multiple redirects are possible in a single url.</li><li> A user id screen is displayed so that the customer can continue trying to purchase.</li></ol> |
```

Don't spell out common acronyms.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • API, REST, POST <br> • File formats (CSV, XML, JSON) <br> • ID, URL, HTML <br> • SKU, EAN <br> • KPI, ROI, GMV <br> • MB, GB <br> • Renew the API token before the duration ends,  | • Megabyte, Gigabyte <br> • Renew the API (Application Programming Interface) token before the duration ends |

Reviewer: ```suggestion
| <ol><li> API, REST, POST</li><li> File formats (CSV, XML, JSON)</li><li> ID, URL, HTML</li><li> SKU, EAN</li><li> KPI, ROI, GMV</li><li> MB, GB</li><li> Renew the API token before the duration ends.</li></ol>  | <ol><li> Megabyte, Gigabyte</li><li> Renew the API (Application Programming Interface) token before the duration ends.</li></ol> |
```

Don't create acronyms from product or feature names.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • Delivery Promise (Beta) is suitable for all types of operations. | • DP (Beta) is suitable for all types of operations |

Reviewer: ```suggestion
| Delivery Promise (Beta) is suitable for all types of operations. | DP (Beta) is suitable for all types of operations |
```

Use *a* or *an based* on pronunciation.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • A URL <br> • A SKU (preferred US pronunciation is /skjuː/)  | • An URL <br> • An SKU  |

Reviewer: ```suggestion
| <ol><li>A URL</li> A SKU (preferred US pronunciation is /skjuː/)</li></ol>  | <ol><li>An URL</li><li> An SKU</li></ol>  |
```

Avoid the possessive form. An alternative is using the preposition *of* using nouns as adjectives, which may require using a definite article or omitting the article.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • The purpose of this FAQ. <br> • This means providing Google with the address of the store website. <br> • The vteximg.com.br domain delivers store image files. | • This FAQ's purpose. <br> • This means providing Google with the address of your store's website. <br> • The vteximg.com.br domain is responsible for delivering your store's image files. |

Reviewer: ```suggestion
| <ol><li>The purpose of this FAQ.</li><li> This means providing Google with the address of the store website. </li><li> The vteximg.com.br domain delivers store image files.</li></ol> | <ol><li>This FAQ's purpose.</li><li> This means providing Google with the address of your store's website.</li><li> The vteximg.com.br domain is responsible for delivering your store's image files.</li></ol> |
```

When making acronyms, initialisms, and other abbreviations plural, treat them as regular words.

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • URL mapping is used to redirect specific URLs to other URLs. <br> • This type of app allows you to create integrations between services using REST APIs or create GraphQL APIs to use data in custom React components. | • URL mapping is used to redirect specific URL to other URL. <br> • This type of app allows you to create integrations between services using REST API’s or create GraphQL APIs to use data in custom React components. |

Reviewer: ```suggestion
| <ol><li>URL mapping is used to redirect specific URLs to other URLs.</li><li> This type of app allows you to create integrations between services using REST APIs or create GraphQL APIs to use data in custom React components.</li></ol> | <ol><li>URL mapping is used to redirect specific URL to other URL.</li><li> This type of app allows you to create integrations between services using REST API’s or create GraphQL APIs to use data in custom React components.</li></ol> |
```

Prefer to use the acronyms of file formats and capitalize them. When specifying file extensions, use lowercase letters (example: \`faststore.config.js\`).

| Do ✅ | Don't ❌ |
| :---- | :---- |

> | • You can set up synonyms individually or by importing a spreadsheet in **CSV** format. <br> • package.**json**: **JSON** file describing the dependencies, script commands for building, and version details. | • You can set up synonyms individually or by importing a spreadsheet in **csv** format. <br> • package.**JSON**: **json** file describing the dependencies, script commands for building, and version details. |

Reviewer: ```suggestion
| <ol><li>You can set up synonyms individually or by importing a spreadsheet in **CSV** format.</li><li> package.**json**: **JSON** file describing the dependencies, script commands for building, and version details.</li></ol> | <ol><li>You can set up synonyms individually or by importing a spreadsheet in **csv** format.</li><li> package.**JSON**: **json** file describing the dependencies, script commands for building, and version details.</li></ol> |
```
